telethon/telegram_client.py

```diff
@@ -2014,3 +2019,30 @@ def get_input_entity(self, peer):
         )
 
     # endregion
+
+
+# region Exceptions
+
+class StopPropagation(Exception):
```

Comment:
This isn't defined inside the `client` instance (which you seem to use in the example above, so `client.StopPropagation` would fail).

Reply:
Oh, you're right. Which way would you prefer?

Reply:
Probably but it in the `errors/` package.